Modules/Magnetometer/fake_magnetometer.py

```diff
@@ -1,6 +1,15 @@
+from ..Field import circular_field
```

Comment:
It seems like too much and i think the sole purpose of ```fake_magnetometer``` is to prevent the program from crashing when running the program without the magnetometer connected.  
It also means that magnetometer depends on the ```Field``` module and that is not optimal in my opinion.

Reply:
But you want is that it will return noly zero's, and we would not be able to work when the magnetometer is not connected